backend/push/src/config/amqp-connection.ts

```diff
@@ -0,0 +1,47 @@
+import { CONFIG } from './config';
+import { connect, Channel, Connection } from 'amqplib';
+import { AmqpQueue } from '~/shared/enums/enums';
+import { logger } from './logger';
+import { timeout } from '../helpers/timeout';
+
+const tryConnect = async (tries = 0): Promise<Connection> => {
+  const amqpServer = CONFIG.rabbitmqUrl;
+  if (tries >= 5) {
+    logger.error('Connection failed');
+    process.exit(0);
```

Comment:
Please not use `process.exit(0)` in this place. You should throw an error and catch it in `backend/push/src/server.ts` file. See the example in `backend/core/server.ts` file